pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -137,12 +139,66 @@ func (p *cadvisorStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 		if vstats, found := p.resourceAnalyzer.GetPodVolumeStats(podUID); found {
 			podStats.VolumeStats = vstats.Volumes
 		}
+		podStats.EphemeralStorage = calEphemeralStorage(podStats.Containers, podStats.VolumeStats, &rootFsInfo)
 		result = append(result, *podStats)
 	}
 
 	return result, nil
 }
 
+func calEphemeralStorage(containers []statsapi.ContainerStats, volumes []statsapi.VolumeStats, rootFsInfo *cadvisorapiv2.FsInfo) *statsapi.FsStats {
```

Comment:
Suggest `calc` or `calculate` instead of `cal`.

Reply:
fixed